db/version_set.cc

```diff
@@ -1457,7 +1457,7 @@ void VersionStorageInfo::SetFinalized() {
     assert(NumLevelBytes(level) == 0);
   }
   uint64_t max_bytes_prev_level = 0;
-  for (int level = base_level(); level < num_levels() - 1; level++) {
+  for (int level = base_level(); level < num_levels(); level++) {
```

Comment:
Maximum bytes are not applicable to the last level. There isn't a further level to compact to, so in theory it is infinite.